### PR TITLE
Adds public pAI cards to North Star

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request

Adds 3 public pAI cards to the map North Star in similar places as commonly found on other maps: The Library, Dormitories, and Science front desk.

## Why It's Good For The Game

This increases the accessibility of pAI cards, particularly towards those without Science, Security, or Medical accesses.

Not having public pAI availability can cause an "out of sight, out of mind" problem - why go through the efforts/costs of obtaining a pAI card when there's no guarantee a personality will be available, and how many people might even think to check? Especially from those that might have looked at a card available at the dormitories or library or so forth, particularly as it takes some time past roundstart for science to be able to print one and the vending machine is priced out of reach of many roles.

In short, I think this change will allow more people to enter rounds on this map, especially at round start, and play more easily.

## Changelog

:cl:
add: Adds pAI cards to public areas on the North Star
/:cl:

### CODE REVIEW

**Comment by Cheshify** [comment]
Yeah sure
